Objects/typeobject.c

```diff
@@ -4888,7 +4888,7 @@ _PyType_GetModuleByDef2(PyTypeObject *left, PyTypeObject *right,
         if (module == NULL) {
             PyErr_Format(
                 PyExc_TypeError,
-                "PyType_GetModuleByDef: No superclass of '%s' and '%s' has"
+                "PyType_GetModuleByDef: No superclass of '%s' or '%s' has"
```

Comment:
Oh, I remember now the conjunction _nor_ should be used for negatives![^1] 

```suggestion
                "PyType_GetModuleByDef: No superclass of '%s' nor '%s' has"
```

[^1]: quoting my dictionary: nor -- used before the second or further of two or more alternatives (the first being introduced by a negative such as ‘neither’ or ‘not’) to indicate that they are each untrue or each do not happen